quickstart/go/helloworld.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+	log.Print("Hello world received a request.")
+	target := os.Getenv("TARGET")
+	if target == "" {
+		target = "World"
+	}
+	fmt.Fprintf(w, "Hello %s!\n", target)
+}
+
+func main() {
+	log.Print("Hello world sample started.")
```

Comment:
I'd delete this and do something like the following right before the call to `log.Fatal` below.

```
log.Printf("Listening on localhost:%s", port)
```